main.py

```diff
@@ -53,15 +75,19 @@ def adopt(listing_id):
     return render_template('adopt.html', listing=listing, form=form)
 
 
-@app.route('/createlisting', methods=['GET', 'POST'])
-def createlisting():
-    form = CreateListingForm()
-    if form.validate_on_submit():
-        f = form.upload_img.data
-        print(f)
-        # TODO: Save the img file into Firebase (?)
-        return redirect(url_for('root'))
-    return render_template('listing.html', form=form)
+# @app.route('/createlisting', methods=['GET', 'POST'])
+# def createlisting():
+#     if request.method == 'POST':
+#         new_listing = Listing()
+#         return Response("", status=201, mimetype='application/json')
+#     if request.method == 'GET':
+#         form = CreateListingForm()
+#         if form.validate_on_submit():
+#             f = form.upload_img.data
+#             print(f)
+#             # TODO: Save the img file into Firebase (?)
+#             return redirect(url_for('root'))
+#         return render_template('listing.html', form=form)
 
```

Comment:
can remove these commented parts cos it's essentially the same as what's in the root function right? so this means we don't need an additional createlisting endpoint.

Reply:
Removed